trackbasket_be/app.py

```diff
@@ -2,7 +2,7 @@
 from flask_restful import Resource, Api
 from flask_sqlalchemy import SQLAlchemy
 from models.volunteer import db, Volunteer
-from resources.volunteer import Volunteer
+from resources.volunteer import Volunteers
```

Comment:
this will help us differentiate models from resources. We could go with singular for models and plural for resources